apps/dashboard/src/components/workflow-editor/steps/conditions/edit-step-conditions-skeleton.tsx

```diff
@@ -0,0 +1,39 @@
+import { RiQuestionLine } from 'react-icons/ri';
+import { Link } from 'react-router-dom';
+import { RiInputField } from 'react-icons/ri';
+
+import { Button } from '@/components/primitives/button';
+import { Panel, PanelContent, PanelHeader } from '@/components/primitives/panel';
+import { Skeleton } from '@/components/primitives/skeleton';
+
+export const EditStepConditionsFormSkeleton = () => {
```

Comment:
I am curious, why do we need a skeleton on step conditions since we have already fetched the workflow with all its steps and controls?

Reply:
yeah, I created it before I realized that it's not needed. Left it for the future if we decide to have separate endpoints.